pkg/operator/podsecurityreadinesscontroller/conditions.go

```diff
@@ -28,24 +32,31 @@ var (
 )
 
 type podSecurityOperatorConditions struct {
-	violatingOpenShiftNamespaces    []string
-	violatingRunLevelZeroNamespaces []string
-	violatingCustomerNamespaces     []string
+	violatingOpenShiftNamespaces      []string
+	violatingRunLevelZeroNamespaces   []string
+	violatingCustomerNamespaces       []string
+	violatingDisabledSyncerNamespaces []string
 }
 
-func (c *podSecurityOperatorConditions) addViolation(name string) {
-	if runLevelZeroNamespaces.Has(name) {
-		c.violatingRunLevelZeroNamespaces = append(c.violatingRunLevelZeroNamespaces, name)
+func (c *podSecurityOperatorConditions) addViolation(ns *corev1.Namespace) {
```

Comment:
Why did you choose to pass `*corev1.Namespace` instead of its name? Curious, since no other fields of the NS are used anywhere.

Reply:
```go
if ns.Labels[labelSyncControlLabel] == "false" {
```

https://github.com/openshift/cluster-kube-apiserver-operator/pull/1686/files#diff-4b300130e48b2b1df6907499643909f1f95fc3eb7064fac655ff704a61695e10R53

Reply:
Ah right, missed that one :+1: